fastddsspy_tool/test/application/launcher.ps1

```diff
@@ -0,0 +1,39 @@
+Param(
```

Comment:
Shouldn't this file have license?

Reply:
Others like this do not have one, I do not really know it